stdlib/pickle.pyi

```diff
@@ -98,7 +98,7 @@ class _ReadableFileobj(Protocol):
     def readline(self) -> bytes: ...
 
 class _WritableFileobj(Protocol):
-    def write(self, __b: bytes) -> Any: ...
+    def write(self, __b: bytes) -> object: ...
```

Comment:
Isn't this Protocol identical to `_typeshed.SupportsWrite[bytes]`?

Reply:
Good catch, removed it